contrastsecurity/datadog_checks/contrastsecurity/contrastsecurity.py

```diff
@@ -0,0 +1,6 @@
+from datadog_checks.base import AgentCheck
+
+
+class ContrastsecurityCheck(AgentCheck):
+    def check(self, instance):
+        pass
```

Comment:
Since this integration does not collect metrics and the implementation is empty you should remove all python code.